general-javascript-best-practices.md

```diff
@@ -145,3 +145,28 @@ var name = "oli";
 var list = [ 1, 4, 5, 7, 9 ];
 doStuff(name, list);
 ```
+
+## Module Template
+
+All modules should follow this order:
+
+1. define a local object and export it
+2. `require()` in any dependencies
+3. (all the logic goes here)
+
+For example:
+```javascript
+// define a local object and export it
+var ourModuleName = module.exports = { };
+
+// Next, require any dependencies:
+var _ = require('underscore');
```

Comment:
Although just an example it would be good to use `lodash` here as we prefer it over `underscore` in our projects now?